From 77fcb26a5088611f463314b7ff45c1822536cdac Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Sat, 16 Oct 2010 22:52:36 +0000
Subject: Hide recovery partition on newer Lenovo machines

Hide the "SYSTEM RESERVED" recovery partition on newer Lenovo
machines.

https://bugs.freedesktop.org/show_bug.cgi?id=30930

Signed-off-by: David Zeuthen <davidz@redhat.com>
---
diff --git a/data/80-udisks.rules b/data/80-udisks.rules
index d1351f3..ef92e85 100644
--- a/data/80-udisks.rules
+++ b/data/80-udisks.rules
@@ -215,7 +215,7 @@ ENV{UDISKS_PARTITION_SCHEME}=="apm", \
 
 # recovery partitions (taken from old hal rules)
 ENV{ID_FS_TYPE}=="ntfs|vfat", \
-  ENV{ID_FS_LABEL}=="RECOVERY|HP_RECOVERY|Recovery Partition|DellUtility|DellRestore|IBM_SERVICE|SERVICEV001|SERVICEV002", \
+  ENV{ID_FS_LABEL}=="RECOVERY|HP_RECOVERY|Recovery Partition|DellUtility|DellRestore|IBM_SERVICE|SERVICEV001|SERVICEV002|SYSTEM RESERVED", \
   ENV{UDISKS_PRESENTATION_HIDE}="1"
 
 # read-only non-Linux software installer partitions
--
cgit v0.8.3-6-g21f6
